doc/users/next_whats_new/EngFormatter-offset.rst

```diff
@@ -0,0 +1,9 @@
+ticker.EngFormatter now computes offset by default
+--------------------------------------------------
+
+``ticker.EngFormatter`` was modified to act very similar to
+``ticker.ScalarFormatter``, such that it computes the best offset of the axis
+data, and shows the offset with the known SI quantity prefixes. To disable this
+new behavior, simply pass ``useOffset=False`` when you instantiate it. If offsets
+are disabled, or if there is no particular offset that fits your axis data, the
+formatter will revert to the old behavior.
```

Comment:
This is also a place where you can throw in a code example via the `.. plot` directive to show off the new feature.

Reply:
> This is also a place where you can throw in a code example via the `.. plot` directive to show off the new feature.

I added an example instead.

Reply:
> you can also copy/paste the example code/simplify it into the what's new - it doesn't need the be distinct, the aim is just to advertise the new functionality

I think it will be better advertised in the examples, but if you think it is not worth an example I won't mind do as you suggested.